Need to specify a working value serializer

No events will be published b/c of lots of warnings like this:

```SHELL
Failed to publish QueryCreatedEvent for query 20210315_013130_00000_kd5ep
org.apache.kafka.common.errors.SerializationException: Can't convert value of class io.trino.spi.eventlistener.QueryCreatedEvent to class org.apache.kafka.common.serialization.ByteArraySerializer specified in value.serializer
Caused by: java.lang.ClassCastException: class io.trino.spi.eventlistener.QueryCreatedEvent cannot be cast to class [B (io.trino.spi.eventlistener.QueryCreatedEvent is in unnamed module of loader 'app'; [B is in module java.base of loader 'bootstrap')
        at org.apache.kafka.common.serialization.ByteArraySerializer.serialize(ByteArraySerializer.java:19)
        at org.apache.kafka.common.serialization.Serializer.serialize(Serializer.java:62)
        at org.apache.kafka.clients.producer.KafkaProducer.doSend(KafkaProducer.java:902)
        at org.apache.kafka.clients.producer.KafkaProducer.send(KafkaProducer.java:862)
        at org.apache.kafka.clients.producer.KafkaProducer.send(KafkaProducer.java:750)
        at io.trino.plugin.eventstream.EventStreamEventListener.queryCreated(EventStreamEventListener.java:43)
        at io.trino.eventlistener.EventListenerManager.queryCreated(EventListenerManager.java:150)
        at io.trino.event.QueryMonitor.queryCreatedEvent(QueryMonitor.java:132)
        at io.trino.dispatcher.LocalDispatchQueryFactory.createDispatchQuery(LocalDispatchQueryFactory.java:120)
        at io.trino.dispatcher.DispatchManager.createQueryInternal(DispatchManager.java:198)
        at io.trino.dispatcher.DispatchManager.lambda$createQuery$0(DispatchManager.java:148)
        at io.trino.$gen.Trino_352____20210315_013051_2.run(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

--- a/README.md
+++ b/README.md
@@ -3,3 +3,21 @@
 A Trino plugin to stream trino events into a Kafka topic.
 
 It implements the [`io.trino.spi.eventlistern.EventListener`](https://github.com/trinodb/trino/blob/f6422d04663ab011d3ddb831ed16dec02659c47e/core/trino-spi/src/main/java/io/trino/spi/eventlistener/EventListener.java) interface.
+
+## Install
+
+Run `mvn install` to build this plugin, then put the plugin file
+`trino-event-stream-352.zip` to the plugin folder of trino server.
+
+## Configuration
+
+Create new properties file `event-listener.properties` inside the `/etc/trino` directory:
+
+```
+event-listener.name=event-stream
+bootstrap.servers=broker:9092
+key.serializer=org.apache.kafka.common.serialization.StringSerializer
+value.serializer=org.apache.kafka.common.serialization.StringSerializer
+```
+
+Then it will emit events to the Kafka topic `trino.event`.

--- a/pom.xml
+++ b/pom.xml
@@ -22,10 +22,15 @@
 	</licenses>
 
 	<properties>
-		<kafka.version>2.5.0</kafka.version>
+		<kafka.version>2.5.1</kafka.version>
 	</properties>
 	
 	<dependencies>
+		<dependency>
+			<groupId>io.airlift</groupId>
+			<artifactId>log</artifactId>
+		</dependency>
+
 		<dependency>
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>

--- a/src/main/java/io/trino/plugin/eventstream/EventStreamEventListener.java
+++ b/src/main/java/io/trino/plugin/eventstream/EventStreamEventListener.java
@@ -43,7 +43,7 @@ public class EventStreamEventListener
         kafkaProducer.send(
                 new ProducerRecord<>(TOPIC_PRESTO_EVENT,
                         queryCreatedEvent.getMetadata().getQueryId(),
-                        queryCreatedEvent));
+                        queryCreatedEvent.toString()));
         log.debug("Sent queryCreated event. query id %s", queryCreatedEvent.getMetadata().getQueryId());
     }
 
@@ -53,7 +53,7 @@ public class EventStreamEventListener
         kafkaProducer.send(
                 new ProducerRecord<>(TOPIC_PRESTO_EVENT,
                         queryCompletedEvent.getMetadata().getQueryId(),
-                        queryCompletedEvent));
+                        queryCompletedEvent.toString()));
         log.debug("Sent queryCompleted event. query id %s", queryCompletedEvent.getMetadata().getQueryId());
     }
 
@@ -63,7 +63,7 @@ public class EventStreamEventListener
         kafkaProducer.send(
                 new ProducerRecord<>(TOPIC_PRESTO_EVENT,
                         splitCompletedEvent.getQueryId(),
-                        splitCompletedEvent));
+                        splitCompletedEvent.toString()));
         log.debug("Sent splitCompleted event. query id %s", splitCompletedEvent.getQueryId());
     }
 }

--- a/src/main/java/io/trino/plugin/eventstream/EventStreamEventListener.java
+++ b/src/main/java/io/trino/plugin/eventstream/EventStreamEventListener.java
@@ -13,6 +13,7 @@
  */
 package io.trino.plugin.eventstream;
 
+import io.airlift.log.Logger;
 import io.trino.spi.eventlistener.EventListener;
 import io.trino.spi.eventlistener.QueryCompletedEvent;
 import io.trino.spi.eventlistener.QueryCreatedEvent;
@@ -26,6 +27,7 @@ import org.apache.kafka.clients.producer.ProducerRecord;
 public class EventStreamEventListener
         implements EventListener
 {
+    private static final Logger log = Logger.get(EventStreamEventListener.class);
     private final KafkaProducer kafkaProducer;
     // TODO make this topic name configurable
     private static final String TOPIC_PRESTO_EVENT = "trino.event";
@@ -42,6 +44,7 @@ public class EventStreamEventListener
                 new ProducerRecord<>(TOPIC_PRESTO_EVENT,
                         queryCreatedEvent.getMetadata().getQueryId(),
                         queryCreatedEvent));
+        log.debug("Sent queryCreated event. query id %s", queryCreatedEvent.getMetadata().getQueryId());
     }
 
     @Override
@@ -51,6 +54,7 @@ public class EventStreamEventListener
                 new ProducerRecord<>(TOPIC_PRESTO_EVENT,
                         queryCompletedEvent.getMetadata().getQueryId(),
                         queryCompletedEvent));
+        log.debug("Sent queryCompleted event. query id %s", queryCompletedEvent.getMetadata().getQueryId());
     }
 
     @Override
@@ -60,5 +64,6 @@ public class EventStreamEventListener
                 new ProducerRecord<>(TOPIC_PRESTO_EVENT,
                         splitCompletedEvent.getQueryId(),
                         splitCompletedEvent));
+        log.debug("Sent splitCompleted event. query id %s", splitCompletedEvent.getQueryId());
     }
 }

--- a/src/main/java/io/trino/plugin/eventstream/EventStreamEventListenerFactory.java
+++ b/src/main/java/io/trino/plugin/eventstream/EventStreamEventListenerFactory.java
@@ -14,6 +14,7 @@
 package io.trino.plugin.eventstream;
 
 import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
 import io.trino.spi.eventlistener.EventListener;
 import io.trino.spi.eventlistener.EventListenerFactory;
 import org.apache.kafka.clients.producer.KafkaProducer;
@@ -24,7 +25,8 @@ import java.util.Map;
 public class EventStreamEventListenerFactory
         implements EventListenerFactory
 {
-    private static final String REGEX_CONFIG_PREFIX = "^event-stream.";
+    private static final Logger log = Logger.get(EventStreamEventListenerFactory.class);
+    // private static final String REGEX_CONFIG_PREFIX = "^event-stream.";
 
     @Override
     public String getName()
@@ -54,9 +56,10 @@ public class EventStreamEventListenerFactory
 
         while (it.hasNext()) {
             String key = it.next();
-            String kafkaConfigKey = key.replaceFirst(REGEX_CONFIG_PREFIX,
-                    "");
-            builder.put(kafkaConfigKey, config.get(key));
+            // String kafkaConfigKey = key.replaceFirst(REGEX_CONFIG_PREFIX,
+            //         "");
+            log.debug("Loading event-listener config %s", key);
+            builder.put(key, config.get(key));
         }
 
         // TODO design ways to config/code serializer

